setup.py

```diff
@@ -49,6 +49,15 @@ def get_version(pkg_name):
     ],
     license="BSD-3-Clause License",
     packages=[PACKAGE_NAME],
-    install_requires=[],
+    install_requires=[
+        "numpy",
+        "scipy",
+        "scikit-image",
+        "scikit-learn",
+        "pandas",
+        "matplotlib",
+        "xarray",
+        "trackpy",
+    ],
```

Comment:
What about `iris` and `cartopy` here? 

And should we maybe add `pytest` as `test_requires` ?

Reply:
See above comment about iris and cartopy. I'm not sure that we can do much until/unless cartopy fixes their pip builds. 

Good catch with `test_requires`, I'll add that shortly.